src/lib.rs

```diff
@@ -313,9 +327,11 @@ where
 /// let mut engine: SimSearch<usize> = SimSearch::new_with(
 ///     SearchOptions::new().case_sensitive(true));
 /// ```
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub struct SearchOptions {
     case_sensitive: bool,
     stop_whitespace: bool,
+    #[cfg_attr(feature = "serde", serde(skip))]
     stop_words: &'static [&'static str],
```

Comment:
`stop_words` can be rewritten in `Vec<String>` so as to support serialization.

Reply:
Ah, that's a much better way of handling it; committed changes